chainer/functions/math/sparse_matmul.py

```diff
@@ -0,0 +1,486 @@
+import chainer
+from chainer.backends import cuda
+from chainer import function_node
+from chainer import utils
+from chainer.utils import type_check
+import numpy
+import warnings
+
+try:
+    from scipy import sparse
+except ImportError:
+    warnings.warn("SciPy seems not available on your system. A CPU"
+                  " implementation of sparse_matmul uses SciPy, so that you"
+                  " cannot use sparse_matmul on CPU.")
+
+
+class sparse_coo_matrix(object):
```

Comment:
Could you rename this class using CamelCase?

Reply:
will use CamelCase for the class name.